### PR TITLE
gsdx-debug: Add secret ini option for dump directory

### PR DESCRIPTION
**Pull request by tadanokojin**
Add a secret ini option to change the raw dump directory.

Also the following changes:
- Windows creates the directory now
- Directory is now `GSDX_[32|64]_[SW|HW]`

#SaveTheSSDs

### CODE REVIEW

File: plugins/GSdx/GSState.cpp [hunk 1]
```diff
@@ -44,6 +44,7 @@ GSState::GSState()
 	, m_crc(0)
 	, m_options(0)
 	, m_frameskip(0)
+	, m_dump_root("")
```
Comment (tadanokojin) [inline plugins/GSdx/GSState.cpp]: my mistake, I don't need to initialize this to an empty string.
will fix tomorrow.

**Comment by gregory38** [comment]
But why not this way too

**Comment by gregory38** [comment]
So. It would be much nicer to have a virtual function that get the root dir.

File: plugins/GSdx/GSdx.cpp [hunk 2]
```diff
@@ -366,6 +366,11 @@ void GSdxApp::Init()
 	m_default_configuration["debug_opengl"]                               = "0";
 	m_default_configuration["disable_hw_gl_draw"]                         = "0";
 	m_default_configuration["dump"]                                       = "0";
+#ifdef _WIN32
+	m_default_configuration["dump_dir"]                                   = "C:\\tmp\\";
```
Comment (lightningterror) [inline plugins/GSdx/GSdx.cpp]: Maybe dumping in install dir is better ? So we go less back and forth.

**Pull request closed without merging**